archetypes/helidon/src/main/archetype/mp/oci/files/server/src/main/java/__pkg__/server/GreetReadinessCheck.java.mustache

```diff
@@ -0,0 +1,41 @@
+package {{package}}.server;
+
+import java.time.Duration;
+import java.util.concurrent.atomic.AtomicLong;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.enterprise.context.Initialized;
+import jakarta.enterprise.event.Observes;
+
+import org.eclipse.microprofile.health.HealthCheck;
+import org.eclipse.microprofile.health.HealthCheckResponse;
+import org.eclipse.microprofile.health.Readiness;
+
+@Readiness
+@ApplicationScoped
+public class GreetReadinessCheck implements HealthCheck {
+    private AtomicLong readyTime = new AtomicLong(0);
```

Comment:
private final; and also remove extra line below

Reply:
fixed.